kis_api.py

```
@@ -1,372 +1,1093 @@
+            response = requests.post(url, headers=headers, json=data, timeout=10)
+            response.raise_for_status()  # HTTP 에러 발생 시 예외 발생
+            return response.json()
+        except Exception:
+            self.logger.exception("직접 API 호출 실패")
+            return None
+    
+    def _direct_api_call_sell(self, symbol, quantity, price):
+        """
+        직접 API 호출로 매도 주문 (mojito2 우회)
+        """
+        import requests
```

Ruff (0.14.1)

Redefinition of unused `requests` from line 7

Remove definition: `requests`

(F811)